packages/sanes-chrome-extension/src/routes/tx-request/components/ShowRequest/ReqRegisterUsernameTx.tsx

```diff
@@ -1,16 +1,72 @@
 import { RegisterUsernameTx } from '@iov/bns';
-import { List } from 'medulas-react-components/lib/components/List';
+import { List, ListItem, ListItemText } from 'medulas-react-components/lib/components/List';
 import * as React from 'react';
+import { makeStyles } from '@material-ui/core';
+import { amountToString } from '../../../../utils/balances';
+import Typography from 'medulas-react-components/lib/components/Typography';
+import Block from 'medulas-react-components/lib/components/Block';
+
+export const REQ_REGISTER_USERNAME = 'req-register-username-tx';
+
+const useStyles = makeStyles({
+  root: {
+    margin: 0,
+  },
+});
+
+const secondaryProps = {
+  noWrap: true,
+};
 
 interface Props {
   readonly creator: string;
   readonly tx: RegisterUsernameTx;
 }
 
-const ReqRegisterUsernameTx = ({ tx, creator }: Props): JSX.Element => {
-  // TODO: implement
-  // https://github.com/iov-one/ponferrada/issues/349
-  return <List />;
+const ReqRegisterUsernameTx = ({ tx }: Props): JSX.Element => {
+  const listItemClasses = useStyles();
+
+  const txFee =
+    tx.fee && tx.fee.tokens ? (
+      <React.Fragment>
+        <Block marginTop={1} />
+        <Typography variant="body1" inline>
+          Fee:{' '}
+        </Typography>
+        <Typography variant="body1" inline>
+          {amountToString(tx.fee.tokens)}
+        </Typography>
```

Comment:
Why two `<Typography>` elements? What about

```tsx
        <Typography variant="body1" inline>
          {'Fee: ' + amountToString(tx.fee.tokens)}
        </Typography>
```

Reply:
Yes, you are right, my mistake. Also `inline` attribute is not required in this case.